core/domain/android_validation_constants.py

```diff
@@ -36,7 +36,7 @@
 # If any of the following values are changed, edit the corresponding value in
 # app.constants.ts as well.
 MAX_CHARS_IN_TOPIC_NAME = 39
-MAX_CHARS_IN_ABBREV_TOPIC_NAME = 12
+MAX_CHARS_IN_ABBREV_TOPIC_NAME = 20
```

Comment:
Just to check -- in a PR which changes URLs, why is this being changed?

Also I was under the impression we had deprecated the abbreviated topic name field ... why is is still around? Where is it used? Maybe check with @mschanteltc? /cc @aks681

Reply:
Initially I hd planned to use abbreviated_name in the URL but later on we decided to use a separate url_fragment field for this. I have reverted this change.

It is not currently used. We have only defined it as a constant in android_validation_constants and in constants (to be in sync).